packaging/antelopeio-dune/antelopeio-dune.nuspec

```diff
@@ -16,10 +15,14 @@
     <projectSourceUrl>https://github.com/AntelopeIO/DUNE</projectSourceUrl>
     <bugTrackerUrl>https://github.com/AntelopeIO/DUNE/issues</bugTrackerUrl>
     <summary>antelopeio-dune</summary>
-    <description>### antelopeio-dune package
+    <description>### antelopeio-dune eos cleos nodeos leap cdt antelope antelopeio
 Docker Utilities for Node Execution (DUNE) is a tool to abstract over Leap programs, CDT and other services/tools related to Antelope blockchains.
     
     </description>
+    <dependencies>
+      <dependency id="python"/>
+      <dependency id="docker-desktop"/>
+    </dependencies>
```

Comment:
In Linux, @jolly-fellow and I found `wget` and `curl` were necessary for a working DUNE instance. I suspect these tools are still required.

Reply:
Actually wget and curl were needed only by packaging scripts for Linux.
So for Windows we do not need them.

Reply:
I have confirmed this as well. LGTM.

Reply:
I believe curl is only used within the docker instance itself in main, so we could legitimately remove it from the deb and rpm dependencies.